deploy-manage/upgrade/deployment-or-cluster/upgrade-717.md

```diff
@@ -90,6 +90,15 @@ This step covers upgrading your deployment from 7.17.x to 8.19.x, following the
 
 It's highly recommended to start this upgrade from the latest 7.17.x patch release to ensure that you’re using the most recent version of the Elastic Upgrade Assistant. You should also upgrade to the latest available 8.19.x patch release so that the same benefits apply when you later upgrade to 9.x.
 
+:::::{note}
+If you use the [{{elastic-sec}} solution](/solutions/security.md), read the full [Upgrade {{elastic-sec}} from 7.17 to 8.x](https://www.elastic.co/guide/en/security/8.19/upgrade-7.17-8x.html) guide when planning the upgrade.
```

Comment:
perhaps this should be a `warning`? 

```suggestion
If you use the [{{elastic-sec}} solution](/solutions/security.md), then you need to perform additional steps as part of the upgrade process. Review [Upgrade {{elastic-sec}} from 7.17 to 8.x](https://www.elastic.co/guide/en/security/8.19/upgrade-7.17-8x.html) before you begin your upgrade.
```

Reply:
I'm in favor of switching the admonition to a warning and using the stronger, more direct language. There's a good amount of pre and post checks that users will need to complete when moving from 7.17->8.x, so the more emphasis, the better.